languages/dart/README.md

```diff
@@ -0,0 +1,170 @@
+# Dart
+
+## Primitives
```

Comment:
You could consider creating a h2 heading named `Types`, and then have the individual types be defined as h3 headings. That way, it is easy to see the type sections go together.